dark-mode.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```

Comment:
remove it if not needed

Reply:
Have updated the code ,and merged all the css files